stories/index.js

```diff
@@ -38,14 +38,14 @@ storiesOf('HexImg', module)
         <div>
             <HexImg
                 url={
-                    'https://penasultra.com/wp-content/plugins/webp-express/test/test-pattern-tv.jpg'
+                    require('./../src/Components/Image/test.jpg')
```

Comment:
This image should be in storybook folder `stories`. Components should not contain any test files.